### PR TITLE
Adding vmware-cs extension to index

### PR DESCRIPTION
**Pull request by shivam-msft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/index.json [hunk 1]
```diff
@@ -2181,6 +2181,49 @@
                 "sha256Digest": "42c6500e5587184026a398147b0a7b1a9cb7e59642de9b932dfcbe1a90f14697"
             }
         ],
+        "vmware-cs": [
+            {
+                "downloadUrl": "https://github.com/Azure/az-vmware-cli/releases/download/v0.1.0/vmware_cs-0.1.0-py2.py3-none-any.whl",
+                "filename": "vmware_cs-0.1.0-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.isPreview": true,
+                    "azext.maxCliCoreVersion": "2.1.0",
```
Comment (zikalino) [inline src/index.json]: you will have to switch sequence of these 2 lines, minCliExtension on the top of maxCliExtension.
Don't ask me why.... but it disrupts validation

File: src/index.json [hunk 2]
```diff
@@ -2181,6 +2181,49 @@
...
+                                "Home": "https://github.com/Azure/az-vmware-cli"
+                            }
+                        }
+                    },
+                    "extras": [],
+                    "generator": "bdist_wheel (0.30.0)",
+                    "license": "MIT",
+                    "metadata_version": "2.0",
+                    "name": "vmware-cs",
+                    "run_requires": [
```
Comment (zikalino) [inline src/index.json]: remove run_requires entirely
do the same in setup.py.
and you will have to rebuild and upload again the wheel

**Comment by zikalino** [comment]
also you will need to add entry to CODEOWNERS

File: src/index.json (hunk 1 above)
Comment (shivam-msft) [reply src/index.json]: Done.

File: src/index.json (hunk 2 above)
Comment (shivam-msft) [reply src/index.json]: Done.

**Pull request merged**